sphinx_audeering_theme/static/css/audeering.css

```diff
@@ -436,3 +436,12 @@ footer li {
 footer p {
   margin-bottom: 10px;
 }
+
+/***** MISCELLANEOUS *****************************************************/
+/* Fixes https://github.com/readthedocs/sphinx_rtd_theme/issues/996 */
+.wy-plain-list-disc li ul,
+.rst-content .section ul li ul,
+.rst-content .toctree-wrapper ul li ul,
+article ul li ul {
+  margin-bottom: 12px;
```

Comment:
I would say that the nested list should have the same bottom and top margin to the higher level as the bottom and top margins of those higher level.

So basically cahnging

![image](https://user-images.githubusercontent.com/173624/167357199-8d568c7f-c5ba-4787-8d08-fca9f1fc8e1f.png)

to

![image](https://user-images.githubusercontent.com/173624/167357377-00a0144d-a16c-430c-b9e4-1fbb2d2d367b.png)

This could be achieved by:
```css
.rst-content .section ol p, .rst-content .section ul p {
  margin-bottom: 0;
}
```
but I'm not sure if this will have some undesired consequences.

Reply:
Hmm, I agree that your suggestion looks better when having a pure list with bullet points. It leads to very condensed text, though, when mixing bullet points with paragraphs, as we do in some places in our docs:

![image](https://user-images.githubusercontent.com/64791786/167400056-358e738a-c9ba-4f9a-9406-9c818bec375d.png)

Compare with my suggestion:

![image](https://user-images.githubusercontent.com/64791786/167399867-1c57f2b0-a935-4c1d-9f34-3e40279e08d1.png)

But both are not looking 100% correct. It might be that we are misusing bullets and paragraphs in some non-standard way. Maybe it makes more sense for us to apply the CSS fix that works for us in our docs directly and leave the audEERING template as it is for now.